.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+# Golang CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-go/ for more details
+version: 2
+jobs:
+  build_and_test:
+    docker:
+      - image: circleci/golang:1.13
```

Comment:
nit:
I think we should use golang officially image like `golang:1.13.8-buster` instead circleci one.

Reply:
Don't have a strong opinion here -- I was just following the CircleCI guide. What's the advantage of the official golang image in comparison to the circleci one?

Reply:
>  What's the advantage of the official golang image in comparison to the circleci one?

Usually, Go official Docker image has been released earlier than CircleCI's one when new version of Go released. 
And also the official image is familiar by Go people than CircleCI's one.
(Though both are not very strong opinion :)

Reply:
Thanks, makes sense! I've changed the image to `golang:1.13.8-buster`, as you suggested. Also checked that the tests still pass.